Analysis/Tasks/PWGHF/HFCandidateCreator2Prong.cxx

```diff
@@ -153,13 +153,14 @@ struct HFCandidateCreator2ProngMC {
     for (auto& candidate : candidates) {
       //Printf("New rec. candidate");
       result = 0;
-
+      auto arrayDaughters = array{candidate.index0_as<aod::BigTracksMC>(), candidate.index1_as<aod::BigTracksMC>()};
       // D0(bar) → π± K∓
       //Printf("Checking D0(bar) → π± K∓");
-      auto indexRecD0 = RecoDecay::getMatchedMCRec(
-        particlesMC, array{candidate.index0_as<aod::BigTracksMC>(), candidate.index1_as<aod::BigTracksMC>()},
-        421, array{+kPiPlus, -kKPlus}, true, &sign);
+      auto indexRecD0 = RecoDecay::getMatchedMCRec(particlesMC, arrayDaughters, 421, array{+kPiPlus, -kKPlus}, true, &sign);
       result += sign * D0ToPiK * int8_t(indexRecD0 > -1);
+      // Jpsi → e+e-
+      auto indexRecJpsi = RecoDecay::getMatchedMCRec(particlesMC, arrayDaughters, 443, array{+kElectron, -kElectron}, true, &sign);
+      result += sign * JpsiToEE * int8_t(indexRecJpsi > -1);
```

Comment:
Please wait with the modifications of this task until https://github.com/AliceO2Group/AliceO2/pull/5224 is merged.

Reply:
It's merged. Please modify this part accordingly.